Added unit tests for amathutils_lib

sgermanserrano opened the pull request:
## Status
**PRODUCTION**

## Description
Unit tests have been added.

## Steps to Test or Reproduce
`catkin_make run_tests_amathutils_lib`

Commit:
Added unit tests for amathutils_lib

sgermanserrano commented:
Tests for autowarefoundation/autoware_ai#373 have been developed

yk-fujii commented:
Thanks

This library will soon be changed to use geometry_msgs::Point 
I think you would better to wait a little.

autowarefoundation/autoware#1609
https://github.com/CPFL/Autoware/blob/6a8c3bcb441dbcadc028fbeb6abf2a62352c40d9/ros/src/common/libs/amathutils_lib/include/amathutils_lib/amathutils.hpp

Commit:
Updated unit tests according to PR #1609 - feature/rebuild_decision_maker

sgermanserrano commented:
@yk-fujii I have updated the unit tests according to the new amathutils and changed this PR from develop to autowarefoundation/autoware#1609 
Further tests need to be added for the new functions in amathutils

s-azumi reviewed (approved):
@sgermanserrano  Thanks!
